src/app/story/storyrow/storyrow.component.html

```diff
@@ -6,6 +6,7 @@
   <ng-template class="node-timestamp" #noTimestamp>
     <div class="node-timestamp">No timestamp available</div>
   </ng-template>
+  <div class="wait-time">{{ waitTime }}</div>
```

Comment:
Why did you decide to make the clock icon and wait time in separate divs? I think it might be easier if they were together.

Reply:
I thought so too, but when I tried it for some reason it screwed with the formatting. I can try again though, maybe with different CSS or something, if you feel strongly about this

Reply:
Could you show me this issue in person or over video chat? We can try to address it together, although I'm not a CSS wizard.  :)

Reply:
What about this comment? Is this something you could try and show me the issue you get?

Reply:
I think I screwed it up on the first attempt because I changed some CSS and now this works, sorry!